ODataConnectedService/src/Templates/ODataT4CodeGenerator.cs

```diff
@@ -413,6 +429,26 @@ public void ValidateAndSetIgnoreUnexpectedElementsAndAttributesFromString(string
     this.IgnoreUnexpectedElementsAndAttributes = boolValue;
 }
 
+/// <summary>
+/// Set the MakeTypesInternal property with the given value.
+/// </summary>
+/// <param name="stringValue">The value to set.</param>
+public void ValidateAndSetMakeTypesInternalFromString(string stringValue)
```

Comment:
`stringValue` Variable type should not form part of the variable name.

Reply:
Other validation methods in that class used the same naming pattern. But you're right I should change it. I'll also then change the existing ones for consistency.

Reply:
Done, replaced with `inputValue`.